scalafix/rules/src/main/scala/fix/Scalacollectioncompat_newcollections.scala

```diff
@@ -500,6 +554,58 @@ case class Scalacollectioncompat_newcollections(index: SemanticdbIndex)
     if (useSites.nonEmpty) useSites + imports
     else Patch.empty
   }
+
+  def replaceBreakout(ctx: RuleCtx): Patch = {
+    import Breakout._
+    
+    def fixIt(intermediate: String, lhs: Term, ap: Term, breakout: Tree): Patch = {
+      ctx.addRight(lhs, "." + intermediate) +
+      ctx.addRight(ap, ".to") +
+      ctx.replaceTree(breakout, "implicitly")
+    }
+
+    ctx.tree.collect {
+      case i: Importee if breakOut.matches(i) =>
+        ctx.removeImportee(i)
+
+      case Term.Apply(ap @ Term.ApplyInfix(lhs, operators(op), _, List(rhs)), List(breakOut(bo))) =>
+        val subject =
+          if(isLeftAssociative(op)) lhs
+          else rhs
+
+        val intermediate = 
+          op match {
+            case operatorsIterator(_) => "iterator"
+            case operatorsView(_)     => "view"
+            case _                    => throw new Exception("impossible")
```

Comment:
Maybe add a comment referring to the definition of `operatorSymbols`, explaining why that makes that path impossible to reach?